package.json

```diff
@@ -101,47 +100,47 @@
   "devDependencies": {
     "@babel/core": "^7.23.2",
     "@babel/preset-env": "^7.23.2",
-    "@commitlint/cli": "^18.6.1",
-    "@commitlint/config-conventional": "^18.6.1",
+    "@commitlint/cli": "^19.2.1",
+    "@commitlint/config-conventional": "^19.1.0",
     "@intlify/vue-i18n-loader": "^5.0.1",
     "@types/lodash-es": "^4.17.7",
-    "@typescript-eslint/eslint-plugin": "^5.61.0",
-    "@typescript-eslint/parser": "^5.61.0",
+    "@typescript-eslint/eslint-plugin": "^7.3.1",
+    "@typescript-eslint/parser": "^7.3.1",
     "@vue/cli-plugin-babel": "~5.0.8",
     "@vue/cli-plugin-e2e-cypress": "~5.0.8",
     "@vue/cli-plugin-eslint": "~5.0.8",
     "@vue/cli-plugin-typescript": "~5.0.8",
     "@vue/cli-plugin-unit-jest": "^5.0.8",
     "@vue/cli-service": "~5.0.8",
     "@vue/compiler-sfc": "^3.3.2",
-    "@vue/eslint-config-airbnb": "^5.3.0",
-    "@vue/eslint-config-typescript": "^10.0.0",
+    "@vue/eslint-config-airbnb": "^8.0.0",
+    "@vue/eslint-config-typescript": "^13.0.0",
     "@vue/test-utils": "^2.4.0",
     "@vue/vue3-jest": "^27.0.0",
     "archiver": "^5.3.1",
     "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.3.0",
+    "babel-loader": "^9.1.3",
     "babel-runtime": "^6.26.0",
-    "chai": "^4.3.7",
     "copy-webpack-plugin": "^8.1.1",
     "core-js": "^3.31.1",
     "cross-env": "^7.0.3",
     "cypress": "^13.6.2",
-    "eslint": "^7.32.0",
+    "eslint": "^8.57.0",
+    "eslint-import-resolver-typescript": "^3.6.1",
```

Comment:
What benefit does this package give us?

Reply:
Without it we will get this type of errors.
```
C:\Users\nikit\Documents\Work\superhero-wallet\tests\unit\transaction-details.spec.js
  2:30  error  Missing file extension for "../../src/constants/stubs"        import/extensions
  2:30  error  Unable to resolve path to module '../../src/constants/stubs'  import/no-unresolved
  ```